instance/static/html/instance/details.html

```diff
@@ -28,6 +28,22 @@ <h3 ng-if="instance">
   </div>
 </div>
 
+<div class="panel" ng-controller="Details">
+  <div class="row">
+    <div class="large-10 columns">
+      <h3>Notes <a ng-click="is_editing_notes=true"><i class="fa fa-edit"></i></a></h3>
+
+      <div ng-if="!is_editing_notes">{{ instance.notes }}</div>
+
+      <form name="editNotesForm" ng-if="is_editing_notes">
+        <textarea rows="4" cols="50" ng-model="instance.notes">{{ instance.notes }}</textarea>
+        <a ng-click="is_editing_notes=false">Cancel</a>
```

Comment:
I also don't know enough angular to know why setting is_editing_notes here doesn't work, but does work in the other place above. Maybe add a `cancel_edit_notes` function to call?

Reply:
That works, but I didn't want to create unnecessary functions if the same could be accomplished using the variable. I'm going to try a little more and if it doesn't work I add that function